tests/run-macros/f-interpolator-neg/Macros_1.scala

```diff
@@ -33,25 +32,25 @@ object Macro {
       private[this] var oldReported = false
       def partError(message : String, index : Int, offset : Int) : Unit = {
         reported = true
-        errors += '{ Tuple5(true, 0, $index, $offset, $message) }
+        errors += '{ Tuple5(true, 0, ${Expr(index)}, ${Expr(offset)}, ${Expr(message)}) }
```

Comment:
For such use cases, it seems we lose a little usability here.

However, from my experience with Scala 3 macros, it is always good to be explicit, as it is so easy to make mistakes.

Reply:
Note that the verbosity in only there because I made sure to not change the semantics in the tests.
Seen this one would immediately try to make it shorter and the natural way would be to write:
```scala
errors += Expr(Tuple5(true, 0, index, offset, message))
```